airbyte-integrations/bases/base-normalization/normalization/transform_catalog/catalog_processor.py

```diff
@@ -178,10 +178,11 @@ def write_yaml_sources_file(self, schema_to_source_tables: Dict[str, Set[str]]):
         Generate the sources.yaml file as described in https://docs.getdbt.com/docs/building-a-dbt-project/using-sources/
         """
         schemas = []
-        for schema in schema_to_source_tables:
+        for entry in sorted(schema_to_source_tables.items(), key=lambda kv: kv[1]):
```

Comment:
Why do these need to be sorted?

Reply:
it's an array of table names, this is to keep it consistent across runs, so they appear in the same order every time in the file and don't cause unnecessary diffs because their position were just inverted randomly